domain_director/domain.py

```diff
@@ -66,16 +65,17 @@ def decide_node_domain(node_id, polygons, lat=None, lon=None, accuracy=None, max
 
     return domain, criteria
 
-
-def get_node_domain(node_id, polygons, wifis=None, api_key="test", lat=None, lon=None, accuracy=None,
+def update_node_domain(node_id, polygons, wifis=None, api_key="test", lat=None, lon=None, accuracy=None,
                     default_domain=None,
-                    max_accuracy=250, treshold_distance=0, switch_time=-1, migrate_only_vpn=False):
+                    max_accuracy=250, treshold_distance=0, default_switch_time=-1, migrate_only_vpn=False):
```

Comment:
underindentend for new func name

Reply:
what exactly do you mean?

Reply:
The parameters of this function call are under-indented.

Reply:
done, i think